RIGS/signals.py

```diff
@@ -102,3 +105,35 @@ def on_revision_commit(sender, instance, created, **kwargs):
 
 
 post_save.connect(on_revision_commit, sender=models.EventAuthorisation)
+
+
+def send_admin_awaiting_approval_email(user, request, **kwargs):
+    # Bit more controlled than just emailing all superusers
+    for admin in models.Profile.objects.filter(email__in=[y for x in settings.ADMINS for y in x]):
```

Comment:
Could make a `admins` `@property` in the `Profile` model. Would make this read a bit better, and require less thought

Reply:
And have the property do the same search I do there?

Reply:
Yeah, this is purely cosmetic in terms of stating intent. This was a very small nitpick - either makes sense!